### PR TITLE
Issues regarding left side panel

### PR DESCRIPTION
**Pull request by JagathNeerukonda**
1. left side panel styling except the scroll bar issue
2. mobile view toggle styling

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/actions-bar/container.jsx [hunk 1]
```diff
@@ -31,10 +33,19 @@ import { withModalMounter } from '/imports/ui/components/modal/service';
...
+const { dataSaving } = Settings;
+const { viewParticipantsWebcams, viewScreenshare } = dataSaving;
 
 export default withModalMounter(withTracker(({ mountModal }) => ({
   amIPresenter: Service.amIPresenter(),
   amIModerator: Service.amIModerator(),
+  presenter: Service.getPresenter(),
+  swapLayout: (getSwapLayout()) && shouldEnableSwapLayout(),
+  usersVideo: VideoService.getAllWebcamUsers(),
+  singleWebcam: (VideoService.getAllWebcamUsers().length < 2),
```
Comment (saiky76) [inline bigbluebutton-html5/imports/ui/components/actions-bar/container.jsx]: why do you need these?

File: bigbluebutton-html5/imports/ui/components/actions-bar/service.js [hunk 2]
```diff
@@ -46,6 +46,7 @@ export default {
     { fields: { presenter: 1 } }).presenter,
   amIModerator: () => Users.findOne({ userId: Auth.userID },
     { fields: { role: 1 } }).role === ROLE_MODERATOR,
+  getPresenter: () => Users.findOne({ presenter: true }),
```
Comment (saiky76) [inline bigbluebutton-html5/imports/ui/components/actions-bar/service.js]: Should you not pass meetingId?

File: bigbluebutton-html5/imports/ui/components/video-provider/component.jsx [hunk 3]
```diff
@@ -906,6 +910,8 @@ class VideoProvider extends Component {
   }
 
   createVideoTag(id, video) {
+    console.log(id, video, 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx');
```
Comment (saiky76) [inline bigbluebutton-html5/imports/ui/components/video-provider/component.jsx]: This needs to be removed

File: bigbluebutton-html5/imports/ui/components/video-provider/video-button/component.jsx [hunk 4]
```diff
@@ -61,6 +61,7 @@ const JoinVideoButton = ({
 
   const disabledLabel = isDisabled
     ? intl.formatMessage(intlMessages.videoLocked) : sharingVideoLabel;
+  console.log(isSharingVideo, '===================>');
```
Comment (saiky76) [inline bigbluebutton-html5/imports/ui/components/video-provider/video-button/component.jsx]: Needs to be removed

File: bigbluebutton-html5/imports/ui/components/actions-bar/container.jsx (hunk 1 above)
Comment (JagathNeerukonda) [reply bigbluebutton-html5/imports/ui/components/actions-bar/container.jsx]: Removed these properties except getPresenter()

File: bigbluebutton-html5/imports/ui/components/actions-bar/service.js (hunk 2 above)
Comment (JagathNeerukonda) [reply bigbluebutton-html5/imports/ui/components/actions-bar/service.js]: Changed the query by including meetingID and online status

File: bigbluebutton-html5/imports/ui/components/video-provider/video-button/component.jsx (hunk 4 above)
Comment (JagathNeerukonda) [reply bigbluebutton-html5/imports/ui/components/video-provider/video-button/component.jsx]: removed unnecessary console statements

**Pull request merged**